.github/workflows/pr.yml

```
@@ -0,0 +1,92 @@
+          echo "Generated:"
+          cat "${{ steps.git-cliff.outputs.changelog }}"
+          echo "Checking..."
+          cat "${{ steps.git-cliff.outputs.changelog }}" | grep -Ev '^(## \[unreleased\]|)$' | grep -q '^.\+$'
+  check-generated-files:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v6
+      - name: Cache
+        uses: Swatinem/rust-cache@v2
+        with: &cache-with
```

actionlint (1.7.12)

[error] anchor "cache-with" is defined but not used

(syntax-check)